lib/MusicBrainz/Server/Controller/WS/js/CheckDuplicates.pm

```diff
@@ -37,11 +37,18 @@ sub check_duplicates : Chained('root') PathPart('check_duplicates') Args(0) {
         $c->detach;
     };
 
+    my @duplicates;
+    if (defined $c->model($model)->search_by_names) {
```

Comment:
That actually calls the method AFAIK (just without any arguments), so a safer way is `$c->model($model)->can('search_by_names')`.